.github/workflows/check_limits.py

```diff
@@ -0,0 +1,26 @@
+def is_Temperature_Ok(temperature):
+    if temperature < 0 or temperature > 45:
+        print("Temperature out of range!")
+        return False
+    return True
+
+def is_Soc_Ok(soc):
+    if soc < 20 or soc > 80:

```

Comment:
if/else range check is repeated in every parameter check  function . It appears that DRY principle violated